tdrs-backend/tdpservice/settings/development.py

```diff
@@ -49,6 +49,3 @@ class Development(Common):
         "all_applications": True,
         "group_models": True,
     }
-
-    # Session Timeout
-    SESSION_TIMEOUT = 5
```

Comment:
Removing this setting causes the development environment to use the timeout in `common.py` which is `30`